addons/point_of_sale/static/tests/tours/ProductConfigurator.tour.js

```diff
@@ -1,84 +1,79 @@
-            ProductConfigurator.do.fillCustomAttribute("Custom Fabric");
-            
+            ProductConfigurator.fillCustomAttribute("Custom Fabric"),
+
             // Confirm configuration
-            ProductConfigurator.do.confirmAttributes();
-            
+            ProductConfigurator.confirmAttributes(),
+
             // Check that the product has been added to the order with correct attributes and price
-            ProductScreen.check.selectedOrderlineHas(
+            ProductScreen.selectedOrderlineHas(
                 "Configurable Chair (Red, Metal, Other: Custom Fabric)",
                 "1.0",
                 "11.0"
-            );
-            
+            ),
+
             // Orderlines with the same attributes should be merged
-            ProductScreen.do.clickHomeCategory();
-            ProductScreen.do.clickDisplayedProduct("Configurable Chair");
-            ProductConfigurator.do.pickColor("Red");
-            ProductConfigurator.do.pickSelect("Metal");
-            ProductConfigurator.do.pickRadio("Other");
-            ProductConfigurator.do.fillCustomAttribute("Custom Fabric");
-            ProductConfigurator.do.confirmAttributes();
-            ProductScreen.check.selectedOrderlineHas(
+            ProductScreen.clickHomeCategory(),
+            ProductScreen.clickDisplayedProduct("Configurable Chair"),
+            ProductConfigurator.pickColor("Red"),
+            ProductConfigurator.pickSelect("Metal"),
+            ProductConfigurator.pickRadio("Other"),
+            ProductConfigurator.fillCustomAttribute("Custom Fabric"),
+            ProductConfigurator.confirmAttributes(),
+            ProductScreen.selectedOrderlineHas(
                 "Configurable Chair (Red, Metal, Other: Custom Fabric)",
                 "2.0",
                 "22.0"
-            );
-            
+            ),
+
             // Orderlines with different attributes shouldn't be merged
-            ProductScreen.do.clickHomeCategory();
-            ProductScreen.do.clickDisplayedProduct("Configurable Chair");
-            ProductConfigurator.do.pickColor("Blue");
-            ProductConfigurator.do.pickSelect("Metal");
-            ProductConfigurator.do.pickRadio("Leather");
-            ProductConfigurator.do.confirmAttributes();
-            ProductScreen.check.selectedOrderlineHas(
+            ProductScreen.clickHomeCategory(),
+            ProductScreen.clickDisplayedProduct("Configurable Chair"),
+            ProductConfigurator.pickColor("Blue"),
+            ProductConfigurator.pickSelect("Metal"),
+            ProductConfigurator.pickRadio("Leather"),
+            ProductConfigurator.confirmAttributes(),
+            ProductScreen.selectedOrderlineHas(
                 "Configurable Chair (Blue, Metal, Leather)",
                 "1.0",
                 "10.0"
-            );
-            return getSteps(); 
-        }
-    });
+            ),
+            Chrome.endTour(),
```

Comment:
Why is `Chrome.endTour()` is added?

Reply:
![image](https://github.com/odoo/odoo/assets/118446179/2cca8081-77ae-4b7f-980a-340832355123)